### ISSUE TITLE
Prosthetic Limbs

### ISSUE THREAD

**Issue by jamesipod** [studio-minus/ppg-bugs-and-requests#254]
**Feature description**
Prosthetic arms and prosthetic legs: When the item is spawned, a full metallic arm (both limb pieces) or leg (all three limb pieces) will appear, with no status bar in detail view. Bringing the respective part near a human missing some portion of the limb will replace the missing portions of the limb (if missing just hand, only the hand part will be prosthetic, if it's the full arm, all of it will be prosthetic. The unused portion of the prosthetic item would be removed on attachment). When they get close enough and become attached, they gain a green status bar. Optionally, a small one-pixel light on each limb of the prosthetic leg and arm could also turn on when attached, to add to the "coolness" factor. They function like normal human limbs, but cannot be stabbed, and do not lose integrity on impacts. Bullets, lasers, heat, or EMPs can disable the limbs (with the red X) and cause them to become limp and have the regular broken machinery overlay until hit with a wrench, or repaired in the context menu. They can still be ripped off of the human, but it takes slightly more force. 

**Feature purpose**
This item would allow us to give limbs to de-limbed humans in a cool way, and repair them while giving them a slight buff. It fits the sci-fi theme of other items in the game.

**Comment by JohnVcav** [comment]
yes, also/or let us spawn normal limbs a pop them back in somehow.
